### ISSUE TITLE
docker run daux generate

### ISSUE THREAD

**Issue by funyx** [dauxio/daux.io#181]
**Describe the bug**
Since this weekend a working daux setup has started failing

```
docker run --rm -w /build -v "$PWD":/build daux/daux.io daux generate -d=htdocs
```

throws 

```
Copying Static assets ...                                             [  OK  ]
Generating ...
- index.html
Fatal error: Uncaught TypeError: Return value of Todaymade\Daux\Tree\Entry::getTitle() must be of the type string, null returned in /daux/libs/Tree/Entry.php:86
Stack trace:
#0 /daux/libs/Tree/Entry.php(184): Todaymade\Daux\Tree\Entry->getTitle()
#1 /daux/libs/Tree/Directory.php(247): Todaymade\Daux\Tree\Entry->dump()
#2 /daux/libs/Config.php(145): Todaymade\Daux\Tree\Directory->dump()
#3 /daux/libs/ContentTypes/Markdown/ContentType.php(55): Todaymade\Daux\Config->getCacheKey()
#4 /daux/libs/Format/Base/ContentPage.php(65): Todaymade\Daux\ContentTypes\Markdown\ContentType->convert('Welcome to the ...', Object(Todaymade\Daux\Tree\Content))
#5 /daux/libs/Format/HTML/ContentPage.php(96): Todaymade\Daux\Format\Base\ContentPage->getPureContent()
#6 /daux/libs/Format/Base/SimplePage.php(20): Todaymade\Daux\Format\HTML\ContentPage->generatePage()
#7 /daux/libs/Format/HTML/Generator.php(183): Todaymade\Daux\Format\Base\SimplePage->getContent()
#8 /daux/libs/Console/RunAction.php(25): Todaymade\Daux\Format\HTML\Generator->Todaymad in /daux/libs/Tree/Entry.php on line 86
```

docs/config.json
```json
{
  "title" : "API Documentation (v0.1)",
  "tagline": "API (v0.1)",
  "image": "assets/logo.png",
  "html": {
    "search": true
  }
}
```

docs/index.md
```md
Welcome to the official API (v0.1).
```

**Comment by onigoetz** [comment]
Hello, the current latest image is currently not working (I'll fix it tonight)

I recommend using the previous release (0.11.1) like this :

```
docker run --rm -w /build -v "$PWD":/build daux/daux.io:0.11.1 daux generate -d=htdocs
```

**Comment by funyx** [comment]
Thanks,
i ll use it as temp fix.. :)
Could you comment here once the latest image is working..

**Comment by onigoetz** [comment]
It's fixed, should be available on docker hub in ~20min

**Comment by funyx** [comment]
```
docker image pull daux/daux.io:latest
```
..and everything works as expected

**Comment by onigoetz** [comment]
The fixes discussed in this issue are available in [version 0.12](https://github.com/dauxio/daux.io/releases/tag/0.12.0)
